### ISSUE TITLE
Checkbox and radio labels shows encoded HTML entities

### ISSUE THREAD

**Issue by giacomovernoni** [livewire/flux#516]
If you copy and paste this snippet in a blade view:
```
@php
    $actor = "Vincent D'Onofrio";
@endphp

<div class="space-y-4">
    <flux:checkbox label="{{ $actor }}" />
    <flux:radio label="{{ $actor }}" />
</div>
```

The output will be:

![Image](https://github.com/user-attachments/assets/91cdedf3-70af-49b2-a5a3-d51edf40ed0e)

I have tested other components (breadcrumb , heading, subheading) and the same string is handled correctly:

![Image](https://github.com/user-attachments/assets/197c6cb2-f308-4289-b8cb-2d1c56cdb939)

**Comment by MiguelLaforest** [comment]
What's you're Flux version, this issue should have been solved in [1.0.11](https://github.com/livewire/flux/releases/tag/v1.0.11) (I had the exact same problem). Otherwise have you tried ```:label="$actor"```, anyway both ways work on my side.

**Comment by giacomovernoni** [comment]
> What's you're Flux version, this issue should have been solved in [1.0.11](https://github.com/livewire/flux/releases/tag/v1.0.11) (I had the exact same problem). Otherwise have you tried `:label="$actor"`, anyway both ways work on my side.

You're right... I thought I had the latest version, but I was still on 1.0.9.

Updated, solved. Closing the issue. Thanks!
